CONTRIBUTING.md

```diff
@@ -1,60 +1,144 @@
 
-1. Fork the repository.
-2. Modify the source. Make sure to focus on the specific change you are contributing. For example, if you also reformat the code, it will be hard for us to focus on your change.
-3. Test the code. Make sure that any local tests pass.
-4. Commit to your fork using clear commit messages.
-5. Create the [pull request](https://github.com/opensearch-project/documentation-website/pulls) and save it, making sure to answer the default questions in the pull request template.
+### Making minor changes
 
-If you need additional support, GitHub provides documentation on [forking a repository](https://help.github.com/articles/fork-a-repo/) and [creating a pull request](https://help.github.com/articles/creating-a-pull-request/).
+If you want to make minor changes to an existing file, you can use this approach:
 
+1. [Fork this repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
 
-## Finding contributions to work on
+1. In your fork on GitHub, navigate to the file that you want to change.
 
-If you’d like to contribute but don't know where to start, try browsing existing issues. Our projects use custom GitHub issue labels for status, version, type of request, etc., but we recommend looking at any issues labeled “good first issue” first. 
+1. In the upper-right corner, select the pencil icon and edit the file.
 
+1. In the upper-right corner, select **Commit changes...***. Enter the commit message and optional description and select **Create a new branch for this commit and start a pull request**.
 
-## Security issue notifications
+### Making major changes
 
-If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security via our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/). Please do **not** create a public github issue.
+If you're adding a new page or making major changes to the documentation, such as adding new images, sections, or styling, we recommend that you work in a local copy of the repository and test the rendered HTML before submitting a PR. 
 
+#### Setting up your local copy of the repository
 
-## Licensing
+Follow these steps to set up your local copy of the repository:
 
-See the [LICENSE](LICENSE) file for our project's licensing. 
+1. [Fork this repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and clone your fork.
+
+1. Navigate to your cloned repository.
+
+1. Install [Ruby](https://www.ruby-lang.org/en/) if you don't already have it. We recommend [RVM](https://rvm.io/), but you can use any method you prefer:
+
+   ```
+   curl -sSL https://get.rvm.io | bash -s stable
+   rvm install 2.6
```

Comment:
There is an issue to update the Ruby version to 2.7: https://github.com/opensearch-project/documentation-website/issues/4498
```suggestion
   rvm install 2.7
```